src/integrations/prefect-dask/pyproject.toml

```diff
@@ -5,8 +5,8 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "prefect-dask"
 dependencies = [
-  "anyio >= 3.7.1, < 4.0.0",
-  "prefect>=2.13.5, < 3.0.0",
+  "exceptiongroup",
+  "prefect>=2.19.9, < 2.19.9",  # This package is not yet compatible with AnyIO 4
```

Comment:
hmm, did we mean for this upper bound to be 3.0.0?

Reply:
Yes, thank you!

Reply:
Fixed